core/types/pow/diff.go

```diff
@@ -10,6 +10,10 @@ import (
 	"math/big"
 )
 
+//45 days c29 will be the main pow graph
+// before 45 days will be c24
+// if 30 s per block
+const BIG_GRAPH_START_HEIGHT = 45 * 1440 * 2
```

Comment:
the 30s block time or other consensus settings should not be hardcoded here, it should be put inside of the `params`